.github/workflows/tests.yml

```
@@ -24,7 +26,7 @@ jobs:
     strategy:
       matrix:
         os: [Ubuntu, MacOS, Windows]
-        python-version: [3.6, 3.7, 3.8, 3.9, "3.10"]
+        python-version: [3.6, 3.7, 3.8, 3.9, "3.10", 3.11-dev]
```

```suggestion
        python-version: [3.6, 3.7, 3.8, 3.9, "3.10", "3.11.0-beta - 3.11.0"]
```
To include pre-releases only

---

`python-version: [3.6, 3.7, 3.8, 3.9, "3.10", "3.11.0-rc - 3.11"]`

---

That is fine. Thanks